statemachine.py

```diff
@@ -44,7 +44,10 @@ def fsm():
         feedback = is_positive(response[-2])    # sort response
         if feedback == -1: #Detect if the user says 'quit'
             break           #Break from loop
-        state = script[state]['next'][feedback]  # switch state of conversation
+        try:
+            state = script[state]['next'][feedback]  # switch state of conversation
+        except:                                 # unsure of specific error which will be thrown - should be added
```

Comment:
```suggestion
        except IndexError:
```
@MarkMain99 Actually, if do it wildly, this could be determined very easily by looking at the error message:
![image](https://user-images.githubusercontent.com/37630203/98055176-d47f8300-1e77-11eb-88ac-50cb9bb5c7e7.png)